### PR TITLE
Add cursor position management for programmed text changes

### PR DESCRIPTION
**Pull request by NunoPinheiro**
When updating the value of the native box, the cursor jumps to position 0 so I needed to add cursor management.

My use case:
 - Some text is inserted on the box
 - In Unity code, I update the box text with a new formatted value
 - Cursor should be kept on the same position

### CODE REVIEW

**Comment by Hemaolle** [comment]
Should we add the same functionality to iOS also?

**Comment by NunoPinheiro** [comment]
If the same behaviour happens there, sure.
I don't need it at the moment, and I am not available to learn objective C today

File: src/androidProj/nativeeditplugin/src/main/java/com/bkmin/android/EditBox.java [hunk 1]
```diff
@@ -380,7 +380,22 @@ private void Remove()
     private void SetText(String newText)
     {
         if (edit != null) {
+            int cursorPos = edit.getSelectionStart();
+            int previousLength = edit.getText().length();
+
             edit.setText(newText);
+
+            //Update text selection (cursor position) to be the same after editing text, if the user had multiple characters selected, we are losing them and get the cursor in only one position
```
Comment (Hemaolle) [inline src/androidProj/nativeeditplugin/src/main/java/com/bkmin/android/EditBox.java]: Could we shorten the lines for the comments an start them with a space? Would be easier to read them on Github for example.

**Comment by Hemaolle** [comment]
Let us now if you need it for iOS at some point

**Pull request merged**